reader.go

```diff
@@ -34,7 +34,7 @@ type Reader struct {
 	file     reader
 	read     reader
 	rowIndex int64
-	values   []Value
+	values   [1]Row
```

Comment:
Following up from https://github.com/segmentio/parquet-go/pull/160/files#r861234029

I'm going to keep it as-is for now because having an array helps avoid creating temporary slices when calling `ReadRows`, will revisit later when we work on optimizing the read path.